webapp/src/components/CalendarPage.tsx

```diff
@@ -1,9 +1,17 @@
+import {
+  Calendar,
+  luxonLocalizer,
+  Event as RBCEvent,
+} from 'react-big-calendar';
 import { Container } from '@mui/material';
+import { DateTime, Settings } from 'luxon';
+import { ProgramWithActivities } from '../types/api';
 import React from 'react';
+import 'react-big-calendar/lib/css/react-big-calendar.css';
```

Comment:
Can we clean up some of these imports so they are sorted and separated between package imports and imports of other files within our webapp, similar to how other files have separated out these files?

Reply:
Great idea!